kotest-framework/kotest-framework-multiplatform-plugin-gradle/src/main/kotlin/io/kotest/framework/multiplatform/gradle/KotestMultiplatformCompilerGradlePlugin.kt

```diff
@@ -17,8 +18,8 @@ class KotestMultiplatformCompilerGradlePlugin : KotlinCompilerPluginSupportPlugi
    companion object {
       const val compilerPluginId = "io.kotest.multiplatform"
       const val KotestGroupId = "io.kotest"
-      const val KotestJsArtifactId = "kotest-framework-multiplatform-plugin-js-jvm"
-      const val KotestNativeArtifactId = "kotest-framework-multiplatform-plugin-native-jvm"
+      const val KotestJsArtifactId = "kotest-framework-multiplatform-plugin-js"
```

Comment:
Not sure why I did -jvm. Is there any drawback to doing -jvm ?

Reply:
> Is there any drawback to doing -jvm?

Yes: for some reason, it breaks the ability to use Gradle's composite builds to replace these artifacts with the local source code.